_data/program.yaml

```diff
@@ -139,11 +151,13 @@ levels:
         url: https://cdn.dirigible.studio/wp-content/uploads/sites/24/2021/09/04073619/TZL-graphic-izl.pdf
       - label: Resonate - Zen and the Way of Making a Difference
         url: https://www.choseizen.org/store/p/resonate
+      - label: 30 minute Zazen
+        url: https://www.youtube.com/watch?v=Qf1vmcP7otk
   - color: black
     label: Shodan
     level: 1
     intention: Wu-Wei
-    edge: Converstional Randori unpacking your reflective essay about your journey
+    edge: Converstional Randori preceded and followed by 30 minute Zazen
```

Comment:
I'm realizing this will be a signature edge form ... we may need to introduce it with an article and video and have the Edge reference it.

It will definitely supplant Ki in Daily Life.